core/server/data/schema/schema.js

```diff
@@ -459,5 +459,16 @@ module.exports = {
         created_by: {type: 'string', maxlength: 24, nullable: false},
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
+    },
+    email_recipients: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        email_id: {type: 'string', maxlength: 24, nullable: false, references: 'emails.id'},
+        member_id: {type: 'string', maxlength: 24, nullable: false, index: true},
+        batch: {type: 'integer', nullable: false, unsigned: true, index: true},
```

Comment:
The indexes here could do with some further performance tests. I added an index to the `batch` column because the background jobs will be fetching with `where email_id="x" and batch=n` but it would be interesting to know if a composite index works better in this scenario. Although I'm not sure how we'd go about adding composite indexes with our schema tools 🤔